genesyscloud/team/resource_genesyscloud_team.go

```diff
@@ -32,56 +32,62 @@ func getAllAuthTeams(ctx context.Context, clientConfig *platformclientv2.Configu
 	if err != nil {
 		return nil, diag.Errorf("Failed to get team: %v", err)
 	}
-
 	for _, team := range *teams {
 		resources[*team.Id] = &resourceExporter.ResourceMeta{Name: *team.Name}
 	}
-
 	return resources, nil
 }
 
 // createTeam is used by the team resource to create Genesys cloud team
 func createTeam(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
 	sdkConfig := meta.(*gcloud.ProviderMeta).ClientConfig
 	proxy := getTeamProxy(sdkConfig)
-
 	team := getTeamFromResourceData(d)
-
 	log.Printf("Creating team %s", *team.Name)
 	teamObj, err := proxy.createTeam(ctx, &team)
 	if err != nil {
 		return diag.Errorf("Failed to create team: %s", err)
 	}
-
 	d.SetId(*teamObj.Id)
 	log.Printf("Created team %s", *teamObj.Id)
+	//adding members to the team
+	members, ok := d.GetOk("member_ids")
+	if ok {
+		memberList := members.([]interface{})
+		//creating members along with teams
+		if len(memberList) > 0 {
+			_, err := proxy.createMembers(ctx, d.Id(), buildTeamMembers(memberList))
+			if err != nil {
+				return diag.Errorf("Failed to create members: %s", err)
+			}
+			log.Printf("Created members %s", d.Id())
+		}
+	}
 	return readTeam(ctx, d, meta)
 }
 
 // readTeam is used by the team resource to read an team from genesys cloud
 func readTeam(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
 	sdkConfig := meta.(*gcloud.ProviderMeta).ClientConfig
 	proxy := getTeamProxy(sdkConfig)
-
 	log.Printf("Reading team %s", d.Id())
-
+	// reading members
+	members, err := readMembers(ctx, d, proxy)
+	if err != nil && members != nil {
+		d.Set("member_ids", members)
+	}
```

Comment:
I think this should be `if err == nil` ? And we should be returning the error if it is not nil

Reply:
addressed